.gitignore

```diff
@@ -22,3 +22,4 @@ thingsboard_gateway/storage/data/
 __pycache__
 /thingsboard_gateway/config/
 /venv/
+thingsboard_gateway/config/logs.conf
```

Comment:
config folder is already in gitignore, probably we don't need to add logs.conf separately.

Reply:
Oops, will remove it =P